ui/components/src/TimeSeriesTooltip/SeriesInfo.tsx

```diff
@@ -23,10 +23,11 @@ export interface SeriesInfoProps {
   markerColor: string;
   totalSeries: number;
   wrapLabels?: boolean;
+  showQuery?: boolean;
 }
 
 export function SeriesInfo(props: SeriesInfoProps) {
-  const { seriesName, formattedY, markerColor, totalSeries, wrapLabels } = props;
+  const { seriesName, formattedY, markerColor, totalSeries, wrapLabels, showQuery = true } = props;
```

Comment:
Added this `showQuery` prop for additional customization of series name display in the tooltip, but I need to expose this on LineChart itself. 

Ideal behavior in the future will probably be to only show the query automatically when more than 1 query was used in the panel.

Reply:
Fixed this up and added a new optional tooltipConfig prop to customize other TimeSeriesTooltip behavior